setup/bundles.rst

```diff
@@ -173,7 +173,7 @@ Before diving into the specifics of the most common edge cases, the general
 recommendation is to **not rely on the Symfony Kernel version** to decide which
 code to use::
 
-    if (Kernel::VERSION_ID <= 20800) {
+    if (Kernel::VERSION_ID < 20800) {
```

Comment:
Yeah, I think you are right. When targetting Symfony 2.8 you will not want to use the deprecated code paths.